flag/flag_test.go

```diff
@@ -8,76 +8,6 @@ import (
 	"github.com/bobuhiro11/gokvm/flag"
 )
 
-func TestParseArg(t *testing.T) {
```

Comment:
The same test is required when using kong.

Reply:
Args/Flags are parsed by kong and I removed the ParseArgs function. Why should this be required?

Reply:
Sorry for the misleading comment. We need to check that the parse by kong is correct.
The cmdline definitions have a direct impact on the user, so it is necessary to keep a stable interface.
So it needs to be covered by an automated test.

I did a quick search and found that the test inside kong was mocking os.args.
Is it possible to achieve the same approach?
https://github.com/alecthomas/kong/blob/master/global_test.go#L15-L18

Reply:
Did have something like this test in mind?

Reply:
Have no clue how to fix the main_test issue at this point.

Reply:
Moved the kong parsing and command execution to flag package.